src/System.Management.Automation/help/HelpCommands.cs

```diff
@@ -180,6 +159,27 @@ public SwitchParameter Examples
         [Parameter(ParameterSetName = "Parameters", Mandatory = true)]
         public string Parameter { set; get; }
 
+        /// <summary>
+        /// List of Component's to search on.
+        /// </summary>
+        /// <value></value>
+        [Parameter]
+        public string[] Component { get; set; } = null;
```

Comment:
Please remove initialization - null is default. Below too.

Reply:
Done.